src/styles/projects-summary.less

```diff
@@ -7,6 +7,21 @@
     padding-right: @grid-gutter-width;
   }
 
+  code {
+    background: transparent;
+    border: none;
+    color: @color-pf-white;
+    white-space: normal;
+  }
+
+  .code-block {
+    background: @color-pf-black;
+    color: @color-pf-black-200;
+    margin-right: 15px;
+    margin-top: 5px;
+    opacity: 0.5;
+  }
+
```

Comment:
Does this change have anything to do with the scrollbars? Can you include a screenshot of how this looks now? @beanh66 should take a look since she designed this piece.